src/main/steps/applicant/task-list/utils.ts

```diff
@@ -8,15 +8,25 @@ export const getKeepYourDetailsPrivateStatus = (userCase: CaseWithId): SectionSt
   if (userCase?.detailsKnown || userCase?.startAlternative) {
     return SectionStatus.IN_PROGRESS;
   }
-  return SectionStatus.NOT_STARTED;
+  return SectionStatus.TO_DO;
 };
 
 export const getConfirmOrEditYourContactDetails = (userCase: CaseWithId): SectionStatus => {
-  if (userCase?.confirmcontactdetails) {
+  if (userCase?.applicant1FullName && userCase?.applicant1DateOfBirth && userCase?.applicant1PlaceOfBirth) {
```

Comment:
why are we only considering 3 attributes for making decision for completion of page? Do we require to consider other fields as well?

Reply:
As discussed, for this jira we considered first three fields only for page completion.
Address & Address History, Email & Phone fields have separate tickets.